clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -172,6 +172,11 @@ New checks
   Detects variables and functions that can be marked as static or moved into
   an anonymous namespace to enforce internal linkage.
 
+- New :doc:`modernize-use-cpp-style-comments
```

Comment:
Please sort this release note correctly after `modernize-min-max-use-initializer-list`